### DISCUSSION TITLE
Filter by user role

### DISCUSSION THREAD

**Discussion by iamkhusainov7** [symfony/symfony#48702] [Symfony Book: The Fast Track]
Hello! I am using PostgreSQL 14+ and symfony 6.2. I am trying to filter users by theirs role, could you please tell me how it can be done? I tried several approaches, but none of them worked :/ As I got the issue is with jsonb roles column. Could any one help me?

**Comment by lyrixx** [comment]
I would set the column in jsonb, then use Json function on it. 

Json operator are not native With doctrine. But there are a few doctrine extension to handle Json function

**Comment by endelwar** [comment]
I wrote a blog post about this a few years ago: https://endelwar.it/2020/08/filter-users-by-role-in-symfony-5/

It also works on Symfony 6 — you'll find the PostgreSQL code in the comments section.
